### PR TITLE
Adding credential-authorization get endpoints

### LINKED ISSUE #2421: Question on API getting access tokens
I'm having trouble finding out how to retrieve all the access tokens for a user other than the currently authenticated user. I'm the enterprise admin for our company and we're trying to automate some checks for personal access tokens with particular scopes.

I'm using my personal access token for this utility. I can find what I need by hitting this URL for a particular USER:

```
URL=https://api.github.com/orgs/${ORG}/credential-authorizations?per_page=10&page=${PAGE}&login=${USER}

curl -s \
    -H "Authorization: token ${TOKEN}" \
    -H "Accept: application/vnd.github.v3+json" \
    "$URL"
```

For convenience, I'd like to use Octokit for the utility but I haven't found how I can do this. Am I missing something obvious?

### PR DESCRIPTION
**Pull request by JonruAlveus**
fixes #2421 
I do not have access to a github enterprise account for testing, so I didn't write any integration tests.

### CODE REVIEW

**Review by nickfloyd — APPROVED** [review]
@JonruAlveus This is another great addition!  Thank you ❤️ - would you mind running the async pagination generators to ensure this and other newly added APIs are covered on the paging front?

`dotnet run --project Octokit.Generators` 

I am planning on integrating this in the build command or even in CI, but I wanted to have it run manually so that we (the humans) could validate that it was doing what it is supposed to do.

**Comment by JonruAlveus** [comment]
> dotnet run --project Octokit.Generators

Welcome back :) Thanks for the review, I've run the update and there appears to be a large diff because the new class has the method in alphabetical order, which it wasn't before.

File: Octokit/Helpers/ApiUrls.cs [hunk 1]
```diff
@@ -4457,5 +4457,15 @@ public static Uri Meta()
         {
             return "meta".FormatUri();
         }
+
+        public static Uri AllOrganizationCredentials(string org)
```
Comment (nickfloyd) [inline Octokit/Helpers/ApiUrls.cs:4461]: Would you also mind adding method comments here?  Sorry I completely missed this during the initial review.

File: Octokit/Helpers/ApiUrls.cs [hunk 2]
```diff
@@ -4457,5 +4457,15 @@ public static Uri Meta()
...
+        {
+            return "orgs/{0}/credential-authorizations".FormatUri(org);
+        }
+
+        public static Uri AllOrganizationCredentials(string org, string login)
```
Comment (nickfloyd) [inline Octokit/Helpers/ApiUrls.cs:4466]: Same as with `public static Uri AllOrganizationCredentials(string org)`

**Comment by nickfloyd** [comment]
@JonruAlveus Let me know if you'd like me to knock these out if you don't have the time.  I'd like to get these changes in for the next release and would be glad to take them over if you are strapped on bandwidth.  I'll need push access to your fork.  Let me know when you get the chance!

**Comment by JonruAlveus** [comment]
@nickfloyd Hi! I’m currently overseas so I’ve added you as a collaborator on my fork. Feel free to knock these out :)

**Comment by nickfloyd** [comment]
release_notes: Adds credential-authorization get endpoints

**Comment by AmandaP87** [comment]
> fixes #2421 I do not have access to a github enterprise account for testing, so I didn't write any integration tests.

U crazy

**Pull request merged**